controllers/opentelemetrycollector_controller.go

```diff
@@ -168,9 +169,12 @@ func (r *OpenTelemetryCollectorReconciler) Reconcile(ctx context.Context, req ct
 func (r *OpenTelemetryCollectorReconciler) RunTasks(ctx context.Context, params reconcile.Params) error {
 	for _, task := range r.tasks {
 		if err := task.Do(ctx, params); err != nil {
-			r.log.Error(err, fmt.Sprintf("failed to reconcile %s", task.Name))
```

Comment:
How often does this logline occur when a pod is terminated?

Reply:
Many. :-).  Like for everything the reconciler touches.  To be honest, it's not that big a deal, and probably something that will only be seen when running kuttle tests.  But I found it very annoying, so I submitted the PR.

Reply:
Would it make sense to end the reconciling with by returing `nil` if the reason is a terminating pod? That way the user gets informed that something did not work and we would get rid of all the unnecessary log lines?